.infrastructure/00_main.tf

```
@@ -0,0 +1,25 @@
+terraform {
+  required_version = "1.2.8"
```

```suggestion
  required_version = "1.5.7"
```
I think this version should be the same as the one specified in the `.terraform-version` file

---

Correct